File: Sorting/InsertionSort/Python/insertionSort.py
def insertionSort(A):
    i = 0
    while(i < len(A)):
        j = i
        while((j > 0) and (A[j-1] > A[j])):
            A[j-1], A[j] = A[j], A[j-1]
            j -= 1
        i += 1

File: Sorting/InsertionSort/Go/insertionSort.go
func insertionSort(A []int) {
    i := 1
    for i < len(A) {
        j := i
        for (j > 0) && (A[j-1] > A[j]) {
            A[j], A[j-1] = A[j-1], A[j]
            j--
        }
        i++
    }
}
